security-privacy-questionnaire.md

```diff
@@ -30,50 +30,55 @@ Additionally, user agents could also choose to persist the last directory a file
 
-No, unless a device exposes such sensors as files or directories. User agents are encouraged to block access to such files or directories (for example `/dev` on linux like systems).
-
-### 2.8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+### 2.8. Do features in this specification enable access to device sensors?
 
-The data this specification lets a user expose to an origin is identical to the data exposed via `<input type=file>` and `<input type=file webkitdirectory>`. The differences are in giving the origins the ability to re-prompt for access to files they previously had access to (if handles were stored in IndexedDB), and in the ability for origins to write back to the files (after explicit permission is granted for that).
+No, unless a device exposes such sensors as files or directories. User agents are encouraged to block access to such files or directories (for example `/dev` on linux like systems).
 
-### 2.9. Does this specification enable new script execution/loading mechanisms?
+### 2.9. Do features in this specification enable new script execution/loading mechanisms?
 
 No.
 
-### 2.10. Does this specification allow an origin to access other devices?
+### 2.10. Do features in this specification allow an origin to access other devices?
 
 Not really. The exception would be devices that are exposed as files or directories by the platform. I.e. network shares or cloud storage sync clients could expose data on other devices in a way that looks like regular files or directories. The user agent could let the user pick these files or directories, thereby giving an origin implicit access to this other device. This API doesn't have any functionality to let a website enumerate all network shares on the local network, only explicitly selected files or directories can be accessed by an origin.
 
-### 2.11. Does this specification allow an origin some measure of control over a user agent’s native UI?
+### 2.11. Do features in this specification allow an origin some measure of control over a user agent’s native UI?
 
 The origin can pop up native file or directory pickers, and have some control over what appears inside that native UI (e.g. accepted file types, starting directory and suggested file names), but that control is very limited. The spec does put limitations on what is allowed as an accepted file type and suggested file name to limit the security impact of allowing websites to control the native UI. User agents are expected to employ similar mechanisms to sanitize the sugessted file names as are used to sanitize for example suggested file names in `<a download="foo.ln">` today.
 
-### 2.12. What temporary identifiers might this this specification create or expose to the web?
+### 2.12. What temporary identifiers do the features in this specification create or expose to the web?
```

Comment:
@alex292 would you like to create a follow-up PR adding a note about https://github.com/WICG/file-system-access/blob/main/proposals/CloudIdentifier.md ?

Reply:
Happy to do so. Given that these identifiers are mostly not *temporary*, is 2.12 the right location for this or should this rather go into 2.5?
Probably also 2.6, right?

Reply:
> Happy to do so. Given that these identifiers are mostly not _temporary_, is 2.12 the right location for this or should this rather go into 2.5?

Good point. Let's do 2.5, since the characteristics of this identifier is up to the CSP's discretion

> Probably also 2.6, right?

Yup, good catch!